packages/suite/src/middlewares/wallet/storageMiddleware.ts

```diff
@@ -124,15 +119,15 @@ const storageMiddleware = (api: MiddlewareAPI<Dispatch, AppState>) => {
                     api.dispatch(storageActions.forgetDevice(action.payload));
                     break;
 
-                case DISCOVERY.UPDATE:
-                case DISCOVERY.INTERRUPT:
-                case DISCOVERY.STOP:
-                case DISCOVERY.COMPLETE: {
+                case discoveryActions.updateDiscovery.type:
+                case discoveryActions.interruptDiscovery.type:
+                case discoveryActions.completeDiscovery.type:
+                case discoveryActions.stopDiscovery.type: {
                     const { deviceState } = action.payload;
```

Comment:
nitpick: can be handled with `isAnyOf` before the switch, but it's already inconsistent, because the same applies to analyticsActions

Reply:
Fixed 

Also refactored it for analytics and message system so it's consistent. Will be larger diff but I believe it's for the greater good 😅

[a802d5f](https://github.com/trezor/trezor-suite/pull/8616/commits/a802d5f07e68c6142a2e8f18da68635a240d397e)